src/main/java/org/broadinstitute/consent/http/db/DataAccessRequestDAO.java

```diff
@@ -104,10 +105,14 @@ WHERE dar.submission_date > now() - interval '1 year'
                 dd.dataset_id
               FROM data_access_request dar
               LEFT JOIN dar_dataset dd ON dd.reference_id = dar.reference_id
-              WHERE dar.submission_date BETWEEN SYMMETRIC :begin AND :end
+              LEFT OUTER JOIN email_entity email ON email.entity_reference_id = dar.reference_id AND email.email_type = :emailType
```

Comment:
Adjusts the query to make it possible to find DARs that are expiring (not before a specific date) and notify them that they're either expiring soon or have expired.